docker/docker.go

```diff
@@ -0,0 +1,131 @@
+package docker
+
+// this package recieves the Docker Hub Automated Build webhook
+// https://docs.docker.com/docker-hub/webhooks/
+// NOT the Docker Trusted Registry webhook
+// https://docs.docker.com/ee/dtr/user/create-and-manage-webhooks/
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	log "github.com/Sirupsen/logrus"
```

Comment:
Can the logging be removed from these changes? since this is a library I encourage returning consistent errors and using an errors package like [this](https://github.com/go-playground/errors) if the errors are more dynamic.